Applications/Terminal/main.cpp

```diff
@@ -136,6 +140,9 @@ int main(int argc, char** argv)
     GFontDatabase::the().for_each_fixed_width_font([&] (const String& font_name) {
         font_menu->add_action(GAction::create(font_name, [&terminal] (const GAction& action) {
             terminal.set_font(GFontDatabase::the().get_by_name(action.text()));
+            auto metadata = GFontDatabase::the().get_metadata_by_name(action.text());
+            terminal.config()->write_entry("Text", "Font", metadata.path);
+            terminal.config()->sync();
```

Comment:
This makes me think that CConfigFile synchronization should probably happen automatically some time after a write_entry.
Let's not worry about it in this patch, just a thought, since it's weird to have to sync() explicitly like this.

Reply:
Yeah, I was thinking earlier about adding a system whereby you pass an integer when you get the config, and if the integer is >= 0 that means you want to autosync at some interval, and if it is negative, it means you don't want autosync. And then just have CConfigFile handle it somehow.